components/SearchBar.js

```diff
@@ -0,0 +1,53 @@
+import { useState } from 'react';
+/* 
+import IconSearch from '../public/img/search.svg'; */
+
+export const SearchBar = () => {
+  const [isSearchOpen, setIsSearchOpen] = useState(false);
+
+  const openSearchBarHandler = () => {
+    isSearchOpen ? setIsSearchOpen(false) : setIsSearchOpen(true);
+  };
+
+  return (
+    <>
+      <div className="flex-initial">
+        <button
+          className="py-1 bg-white md:hidden"
+          onClick={openSearchBarHandler}
+        >
+          <span className="sr-only">Vyhledat</span>
+          {/*  <IconSearch /> */}
+          hledej
+        </button>
+
+        <form
+          className={` ${
+            isSearchOpen || 'hidden'
```

Comment:
I am not sure, if tailwind does some magic on top of it. But be careful that any weird values like `false`/`undefined`/`null`/`0` are not getting printed into the classname. In this case anything "truthy"

Which is the case
![CleanShot 2022-03-22 at 20 02 40](https://user-images.githubusercontent.com/12488648/159556323-4fe98f79-01bf-4c2f-94c8-efc2732431fb.png)

You have the same conditional logic in classnames on multiple places. So you either can approach it carefully and/or use ternary.

Or I would recommend going for this small library - https://github.com/JedWatson/classnames

Reply:
OK I installed the library